edenai_apis/features/llm/llm_moderation_mixin.py

```
@@ -0,0 +1,52 @@
+
+
+class LlmModerationMixin:
+    """Default LLM Moderation implementation that delegates to self.llm_client."""
+
+    def _get_llm_client(self):
+        """Get the LLM client. Override in subclasses if needed."""
+        return self.llm_client
+
+    def llm__moderation(
+        self,
+        input: Union[str, List[Union[str, Dict]]],
```

Ruff (0.15.7)

[error] Function argument `input` is shadowing a Python builtin

(A002)